Source/WebCore/editing/CompositeEditCommand.cpp

```diff
@@ -1608,13 +1612,13 @@ bool CompositeEditCommand::breakOutOfEmptyListItem()
 
     RefPtr<Element> newBlock;
     if (RefPtr blockEnclosingList = listNode->parentNode()) {
-        if (is<HTMLLIElement>(*blockEnclosingList)) { // listNode is inside another list item
+        if (RefPtr liElement = dynamicDowncast<HTMLLIElement>(*blockEnclosingList)) { // listNode is inside another list item
```

Comment:
The style checker is wrong: it does not consider this conditional to use a brace.

Reply:
Maybe the comment is what confuses it?

Reply:
Makes sense. I'll leave that as-is for now.